vault/templates/vault/browse.html

```diff
@@ -63,6 +63,62 @@ <h5 class="card-title float-start">
     </div>
 </div>
 
+<div class="modal confirm-read-permissions" tabindex="-1" id="confirmRevokeReadPermissions">
+  <div class="modal-dialog modal-md">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title">Revoke Read Permissions</h5>
+        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+      </div>
+      <div class="modal-body">
+        <p>
+            Are you sure you would like to revoke read permissions of members
+            of this research group to this vault package?
+        </p>
+        <p>
+            This will revoke read permissions to this package for all normal and read-only members
```

Comment:
Should I say "for all non-datamanager members" or simply "for all members"? I know the datamanagers aren't actually members of the research group but it might still be confusing?

Reply:
I think 'for all members' is the clearest option